docs/sources/auth/auth-proxy.md

```diff
@@ -293,6 +293,33 @@ With this, the user `leonard` will be automatically placed into the Loki team as
 
 [Learn more about Team Sync]({{< relref "team-sync.md" >}})
 
+## Organization roles
+Roles can be set via the headers configuration. The specific attribute to support this feature is called 'Orgs'. e.g. 
+
+```bash
+# Optionally define more headers to sync other user attributes
+headers: 'Orgs:X-WEBAUTH-ORGS'
+```
+
+The value is something in the lines of:
+
+```bash
+X-WEBAUTH-ORGS: 1,2,3
+```
+or 
+```bash
+X-WEBAUTH-ORGS: 1:Admin,2:Viewer,3:Editor
+```
+or anything in between
+```bash
+X-WEBAUTH-ORGS: 1:Admin,2,3:Editor
+```
+
+Where 1, 2 & 3 are the IDs of existing organizations. Behind the (optional) colon (:), you can put the role assigned to this organization. You can only choose between Admin, Editor or Viewer.
+If nothing is provided or a typo is made (it is case-sensitive as well), it will default to having only viewing rights for that organization.
```

Comment:
```suggestion
If nothing is provided or a typo is made (it is case-sensitive as well), then it will default to having only viewing rights for that organization.
```